spec/models/beer_spec.rb

```diff
@@ -0,0 +1,59 @@
+require 'rails_helper'
+
+RSpec.describe Beer, type: :model do
+  let(:beer) { Beer.create(name: "Titan", 
+                        state: true,
+                        description: "aggressively hopped ipa", 
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Trailing whitespace detected.